server/index.js

```diff
@@ -0,0 +1,99 @@
+const express = require("express")
+const path = require('path')
+const fs = require('fs')
+const match = require('path-to-regexp').match
+const fetch = require('node-fetch')
+const composeHeadData = require('./util').composeHeadData
+const composeJSONLD = require('./util').composeJSONLD
+
+const API_BASE_URL = process.env.REACT_APP_API_BASE_URL || 'http://localhost:2503'
```

Comment:
we shouldn't use `:2503` as the default port here, that's the qri API server route. how about defaulting to `:3000`? That's what the frontend HMR server runs on